filesystem/src/main/java/com/nytimes/android/external/fs/FSReader.java

```diff
@@ -51,4 +53,20 @@ public void call(Emitter<BufferedSource> emitter) {
             }
         }, Emitter.BackpressureMode.NONE);
     }
+
+
+    public Observable<BufferedSource>  readAll(@Nonnull final String type) throws FileNotFoundException {
+        return Observable
+                .from(fileSystem.list(type))
```

Comment:
good call! running around at a few conference until next week. Will land and fix then.